src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_pt.properties

```diff
@@ -44,7 +44,8 @@ overload.methods.declaration=Métodos de sobrecarga não devem ser separados. O
 parameter.assignment=A atribuição ao parâmetro ''{0}'' não é permitda.
 require.this.method=A chamada de método no ''{0}'' precisa de \"{1}this.\".
 require.this.variable=A referência à variável de instância ''{0}'' precisa de \"{1}this.\".
-return.count=O número de \"return\"s é {0,number,integer} (o máximo permitido é {1,number,integer}).
+return.count=O número de \"return\"s é {0,number,integer} (o máximo permitido para métodos não vazios/lambdas é {1,number,integer}).
+return.countVoid=O número de \"return\"s é {0,number,integer} (o máximo permitido para métodos vazios/construtores/métodos/lambdas é {1,number,integer}).
```

Comment:
`métodos` is used twice, please recheck all translations for duplicating `methods`

Reply:
done